library/consul

```diff
@@ -1,6 +1,7 @@
 # maintainer: James Phillips <james@hashicorp.com> (@slackpad)
 
-latest: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
+latest: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
+0.7.1: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
 0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
 v0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
```

Comment:
Actually misread that - should I drop all the old versions other than 0.7.1? We don't intend to update any of the previous ones at this point.